reference/v8.json

```diff
@@ -607,6 +608,11 @@
           "js": "0.10.0",
           "ios": "2.0.0",
           "android": "2.0.1"
+        },
+        "auto": {
+          "js": "TBD",
```

Comment:
@lucaswoj 296d335 makes support for all SDKs for undefined for `auto` for now. I'm guessing about how a style spec change this should all be orchestrated to the SDKs so please let me know when we should add the specific version (at least on the GL JS side). 

https://github.com/mapbox/mapbox-gl-js/pull/3203 has added support for `auto` to GL JS but still needs review and the tests in https://github.com/mapbox/mapbox-gl-test-suite/pull/144